### PR TITLE
added section for explaining stack choice of talawa (updated)

### LINKED ISSUE #450: Feature Request: Add to "Applications" section
**Is your feature request related to a problem? Please describe.**
Our applications page is missing detail on why we chose typescript, flutter, and monogodb. 

**Describe the solution you'd like**
Add to the "Applications" page information on why we used typescript, flutter, and monogodb. Our youtube page has videos that could be a helpful resource: https://www.youtube.com/@PalisadoesOrganization/videos. 

**Describe alternatives you've considered**
N/A

**Approach to be followed (optional)**
N/A

**Additional context**
N/A

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by SalilHarit**
What kind of change does this PR introduce?

Added to the "Applications" page information on why talawa uses typescript, flutter, and monogodb in their projects

Issue Number:

Fixes https://github.com/PalisadoesFoundation/talawa-docs/issues/450

Did you add tests for your changes?

No

**Snapshots/Videos:**

**If relevant, did you update the documentation?**

**Summary**

**Does this PR introduce a breaking change?**

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-docs/blob/develop/CONTRIBUTING.md)?**

yes

### CODE REVIEW

**Comment by Cioppolo14** [comment]
@lonereaper Please update to the latest version of Docusaurus and remove yarn.log.  I believe that is why the tests are failing. Thank you.

**Comment by lonereaper** [comment]
i have updated Docusaurus and removed yarn.log from my fork is it working now ?

> @lonereaper Please update to the latest version of Docusaurus and remove yarn.log. I believe that is why the tests are failing. Thank you.

**Comment by SalilHarit** [comment]
i fixed it 

> @lonereaper Please update to the latest version of Docusaurus and remove yarn.log. I believe that is why the tests are failing. Thank you.

**Comment by noman2002** [comment]
@lonereaper Please fix the failing test.

File: .gitignore [hunk 1]
```diff
@@ -24,3 +24,5 @@ package-lock.json
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+yarn.lock*
```
Comment (palisadoes) [inline .gitignore]: This absolutely needs to remain. Remove this immediately or else Docusaurus will fail to run for everyone else. Revert this. You need to upgrade to the latest version of Docusaurus.

**Review by palisadoes — CHANGES_REQUESTED** [review]
The only file that should be submitted are markdown files. No other files should be included. Please remove  `.gitignore` and `yarn.lock` from the PR.

**Review by palisadoes — CHANGES_REQUESTED** [review]
Only the markdown file should get in there PR. Delete all the other ones. They are out of scope

**Comment by CyberWake** [comment]
@lonereaper please resolve the conflicts

**Comment by Cioppolo14** [comment]
@lonereaper Can you address why it is showing over 5000 files changed?

**Comment by SalilHarit** [comment]
> @lonereaper Can you address why it is showing over 5000 files changed?

i was asked to remove .gitignore and then vscode pushed the nodemodules to commit because of it

**Pull request closed without merging**